project/AppDependencies.scala

```diff
@@ -22,10 +22,10 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"   %% "bootstrap-play-26" % "1.16.0",
+    "uk.gov.hmrc"   %% "bootstrap-play-26" % "2.2.0",
     "uk.gov.hmrc"   %% "domain"            % "5.9.0-play-26",
```

Comment:
bumped to 5.10.0-play-26 which can be found [here](https://github.com/hmrc/domain/releases/tag/v5.10.0)